crates/ruff_linter/resources/test/fixtures/flake8_implicit_str_concat/ISC.py

```diff
@@ -91,3 +91,39 @@
 _ = "\12""8" # fix should be "\128"
 _ = "\12""foo" # fix should be "\12foo"
 _ = "\12" ""  # fix should be "\12"
+
```

Comment:
Could you add an example where there are multiple adds: `"a" + "b" + "c" + "d" + "e"`